### DISCUSSION TITLE
Crysol

### DISCUSSION THREAD

**Discussion by franke-biosaxs** [biosaxs-com/atsas-community#622] [Model evaluation and manipulation]
On 2011-06-04 14:17:21, user ashar wrote:

Hi,

New user .. short of time ...

Maybe this question is listed in some other feed on this forum so i am sorry to post this again ...

I have crysol ... i have seen it in functional detail at a research group where i was occupied with other forms of processing of data to get into details of how crysol works.

I know the basics on x-ray scattering ..
What i understand is if you pass a structure to crysol in one of its accepted fomats it generates a corresponding spectra ...

My problem is that i want to generate the spectra of a structure (macro-molecule) i have all the co-ordinates of the monomers ... what i dont have is the hydration shell (a question here would be is it absolutely necessary for crysol to have the hydration shell or can it calculate the spectra without it as well)... the macromolecule is artificial (i made it up, consider that the structure i have is in a vaccum) ... now i want to generate its spectra ... since i dont know how crysol calculates the scattering i just need a brief intro about the parameters that crysol requests when it starts up ...

so in short ... i have a structure ... i need to calculate its x-ray spectra .. how should i do it?

Best
/A

**Comment by franke-biosaxs** [comment]
On 2011-06-08 19:08:43, user Alex wrote:

> My problem is that i want to generate the spectra of a structure (macro-molecule) i have all the co-ordinates of the monomers ... what i dont have is the hydration shell (a question here would be is it absolutely necessary for crysol to have the hydration shell or can it calculate the spectra without it as well)... the macromolecule is artificial (i made it up, consider that the structure i have is in a vaccum) ... now i want to generate its spectra ... since i dont know how crysol calculates the scattering i just need a brief intro about the parameters that crysol requests when it starts up ...

if you run "CRYSOL PDBFILE", it will use default value for the density of the
hydration shell. If you have experimental data, CRYSOL will adjust within certain (experimentally found) limits the density of hydration shell(HS) in order to fit the data.
Default value is applicable for most of the cases. If you have some prior knowloedge about density of HS, you can run "CRYSOL PDBFILE /dro X",
where X is a contrast of a hydration shell.

HTH
ALex
